activities/Abecedarium.activity/play.js

```diff
@@ -313,7 +313,7 @@ enyo.kind({
 		this.$.colorBar.removeClass("themeColor"+this.theme);
 		this.$.colorBar.addClass("themeColor-1");
 		var playMode = this;
-		requirejs(["webL10n"], function(webL10n) {
+		requirejs(["webL10n", "l10n"], function(webL10n,l10n) {
```

Comment:
I don't understand the idea here?

Reply:
I actually did the change(took it literally,sorry) that you suggested in the previous commit, will rectify it soon